src/content/docs/en/guides/upgrade-to/v5.mdx

```diff
@@ -582,3 +582,67 @@ Know a good resource for Astro v4.0? [Edit this page](https://github.com/withast
 ## Known Issues
 
 Please check [Astro's issues on GitHub](https://github.com/withastro/astro/issues/) for any reported issues, or to file an issue yourself.
+
+
+### [changed]: `entryPoint` type inside the hook `astro:build:ssr`
+In Astro v4.x, the `entryPoint` type was `RouteData`.
```

Comment:
I'm looking at the existing docs, and I can only find an entry for `entryPoints` (plural) in that hook -- is this that one? Is one of them wrong?

Reply:
That's a typo, yes